first-class-patterns.cabal

```diff
@@ -47,19 +47,20 @@ library
                        Data.Pattern.Base.Tuple
                        Data.Pattern.Common
   Other-modules:       Data.Pattern.Base.Difference
-  Default-extensions:  TypeFamilies,
-                       GADTs,
-                       ScopedTypeVariables,
-                       TypeOperators,
-                       EmptyDataDecls,
-                       Rank2Types,
-                       GeneralizedNewtypeDeriving,
+  Default-extensions:  TypeFamilies
+                       GADTs
+                       ScopedTypeVariables
+                       TypeOperators
+                       EmptyDataDecls
+                       Rank2Types
+                       GeneralizedNewtypeDeriving
```

Comment:
You can also make it in alphabetic order if we are doing clean up 🙂

Reply:
Good point, @vrom911 🙂 I don't want to refactor whole `.cabal` file at once. But I can do lazy cleanups, prettifying parts of it step by step.